File: switch_or_match_statement/Exercise 1/switch.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter a day number (1-7): ", (day) => {
switch (day.trim()) {
    case "1":
        console.log("Monday");
    break;
    case "2":
        console.log("Tuesday");
    break;
    case "3":
        console.log("Wednesday");
    break;
    case "4":
        console.log("Thursday");
    break;
    case "5":
    console.log("Friday");
    break;
    case "6":
        console.log("Saturday");
    break;
    case "7":
        console.log("Sunday");
        break;
    default:
        console.log("You entered an invalid number");
}
rl.close();
});

File: switch_or_match_statement/Exercise 1/switch.cs
Console.Write("Enter a day number (1-7) : ");
int day = int.Parse(Console.ReadLine()!);
switch (day)
{
    case 1:
        Console.Write("Monday");
        break;
    case 2:
        Console.Write("Tuesday");
        break;
    case 3:
        Console.Write("Wednesday");
        break;
    case 4:
        Console.Write("Thursday");
        break;
    case 5:
        Console.Write("Friday");
        break;
    case 6:
        Console.Write("Saturday");
        break;
    case 7:
        Console.Write("Sunday");
        break;
    Default:
        Console.Write("you entered an invalid number");
}